SAGA/observing/__init__.py

```diff
@@ -0,0 +1,5 @@
+"""
+Contains helper functions and utilities for SAGA observing runs
+"""
+
+from .aat import *
```

Comment:
Is this intended? I can imagine that if at a later time we add `observing.mmt`, there may be name clashes if we import * from both aat and mmt?

Reply:
It was intended in that I'm curating the list that comes from `aat` using `__all__`.  I see your point about possible name clashes, though.  Maybe it should be `from . import aat` and then be used as `from saga.observing.aat import ...`?

Reply:
Yes, I think it is more robust for the users to do `from saga.observing.aat import ... ` or `from saga.observing import aat`. 

Note that for these use cases, we only need an empty `__init__.py` since `aat` itself is a submodule of `saga.observing`.